CleverPush/Source/CleverPushInstance.m

```diff
@@ -1773,6 +1775,27 @@ - (void)handleNotificationOpened:(NSDictionary* _Nullable)payload isActive:(BOOL
     handleNotificationOpened(result);
 }
 
+- (void)handleSilentNotificationReceivedWithAppBanner:(NSDictionary* _Nullable)messageDict {
+    NSDictionary* notification = [messageDict cleverPushDictionaryForKey:@"notification"];
+
+    if (!notification) {
+        return;
+    }
+
+    NSString* notificationId = [notification cleverPushStringForKey:@"_id"];
+
+    if ([CPUtils isEmpty:notificationId] && [notificationId isEqualToString:@"chat"]) {
```

Comment:
This condition is never true:
```[CPUtils isEmpty:notificationId] && [notificationId isEqualToString:@"chat"]```

Reply:
do we even need it?

Reply:
Yes, it will not be true.
We can just keep the below thing.

```
if ([CPUtils isEmpty:notificationId]) {
        return;
 }
```